src/pymovements/utils/parsing.py

```diff
@@ -19,9 +19,9 @@
 # SOFTWARE.
 """Module for parsing input data.
 
-.. deprecated:: v0.21.0
+.. deprecated:: v0.22.0
    Please use gaze.from_asc() instead of utils.parsing.
-   This module will be removed in v0.26.0.
+   This module will be removed in v0.27.0.
```

Comment:
I think this is the third change to `utils/parsing` I'm seeing in PRs submitted within a day. is there any story to it?